server/api/appointments/getTimes.php

```diff
@@ -125,16 +125,30 @@ function calculateRemainingAppointmentsAvailableForResidentialAppointment($appoi
 	return ceil($availableAppointmentSpots) - $appointmentCount;
 }
 
-function calculateRemainingAppointmentsAvailableForInternationalAppointment($treatyType, $numberAppointmentsAlreadyScheduled) {
-	if ($treatyType === 'china') {
-		return 15 - $numberAppointmentsAlreadyScheduled;
-	} else if ($treatyType === 'india') {
-		return 6 - $numberAppointmentsAlreadyScheduled;
-	} else if ($treatyType === 'treaty') {
-		return 5 - $numberAppointmentsAlreadyScheduled;
-	} else if ($treatyType === 'non-treaty') {
-		return 15 - $numberAppointmentsAlreadyScheduled;
-	} 
+function calculateRemainingAppointmentsAvailableForInternationalAppointment($treatyType, $numberAppointmentsAlreadyScheduled, $appointmentTimeId) {
+	// TODO: This is a real band-aid that we are using to enable less appointments for certain time slots
+	$appointmentTimeIdsForLessAppointments = array(968, 969, 970, 971, 972);
+	if (in_array($appointmentTimeId, $appointmentTimeIdsForLessAppointments)) {
+		if ($treatyType === 'china') {
+			return 7 - $numberAppointmentsAlreadyScheduled;
```

Comment:
Do you care if these return negative numbers?

Reply:
Yah that's fine--the FE won't let clients schedule into times that are negative, and it actually informs a logged-in volunteer that the time is "overscheduled by X appointments"